test/cluster/cassandra/docker-compose.yml

```diff
@@ -10,12 +10,12 @@ networks:
         - subnet: 172.42.0.0/16
 services:
   cassandra1:
-    image: cassandra
+    image: cassandra:5.0-beta1
     healthcheck:
```

Comment:
I'm unsure if we are OK with testing only with newest beta cassandra instead of stable. WDYT @Lorak-mmk?

Reply:
We can use this in addition to stable, but not instead of stable

Reply:
I've moved cassandra-5.0-beta1 into a separate compose file.